Source/Core/AxisAlignedBoundingBox.js

```diff
@@ -191,6 +193,18 @@ define([
         return Intersect.INTERSECTING;
     };
 
+    /**
+     * @private
+     */
+    AxisAlignedBoundingBox.closestLocationIn = function(position, boundingObject) {
```

Comment:
I would probably rename these functions as `projectPoint` or something along those lines.  We are essentially projecting the position onto the bounding volume, right?